cmd/fossa/cmd/test/e2e_test.go

```diff
@@ -0,0 +1,148 @@
+package test_test
+
+import (
+	"encoding/json"
+	"flag"
+	"math/rand"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/apex/log"
+	"github.com/stretchr/testify/assert"
+	"github.com/urfave/cli"
+
+	"github.com/fossas/fossa-cli/api/fossa"
+	"github.com/fossas/fossa-cli/cmd/fossa/cmd/test"
+)
+
+var orgID = rand.Intn(1000)
```

Comment:
[nit] Let's bump this to `2^32-1`.

Reply:
@liftM why is making this random good or bad? I'd prefer having static state (see my object mother example above).

Reply:
resolved in person